modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/viewmodel/DoneViewModel.kt

```diff
@@ -37,6 +36,20 @@ class DoneViewModel @Inject constructor(
         analyticsTracker.track(AnalyticsEvent.ACCOUNT_UPDATED_DISMISSED)
     }
 
+    sealed class State {
+        data object Empty : State()
+        data class SuccessFullChangedEmail(
+            var detail: String = "",
+            val tittleResourceId: Int = R.string.profile_email_address_changed,
```

Comment:
Nitpick: Typo in `tittleResourceId` in both `SuccessFullChangedEmail` and `SuccessFullChangedPassword`.